### PR TITLE
doc: fix factual inaccuracy in doc/architecture.rst

### PR DESCRIPTION
**Pull request by smithfarm**
Signed-off-by: Sage Weil <sage@redhat.com>
Signed-off-by: Nathan Cutler <ncutler@suse.com>

### CODE REVIEW

**Comment by smithfarm** [comment]
@liewegas I wasn't sure which of these two statements is inaccurate: "The Ceph Monitor can ping a Ceph OSD Daemon periodically to ensure that it is running" or "Ceph also empowers Ceph OSD Daemons to determine if a neighboring OSD is down, to update the cluster map and to report it to the Ceph monitor(s)" (or both?)

**Comment by smithfarm** [comment]
This relates to @liewegas 's recent email to ceph-devel mailing list, Subject: Re: Which function is mon ping osd?

File: doc/architecture.rst [hunk 1]
```diff
@@ -360,21 +360,23 @@ ability to leverage this computing power leads to several major benefits:
...
+   Daemon cannot notify the Ceph Monitor that it is ``down``. The OSDs
+   periodically send messages to the mon (``pg stats`` pre-luminous, and a new
+   ``MOSDBeacon`` in luminous).  If the Ceph Monitor doesn't see that message
+   after a configurable period of time then it marks the OSD down. However,
+   Ceph also empowers Ceph OSD Daemons to determine if a neighboring OSD is
+   ``down``, to update the cluster map and to report it to the Ceph Monitor(s).
+   This means that Ceph Monitors can remain lightweight processes.  See
+   `Monitoring OSDs`_ and `Heartbeats`_ for additional details.
+
+#. **Data Scrubbing:** As part of maintaining data consistency and cleanliness,
```
Comment (tchaikov) [inline doc/architecture.rst]: weird, i cannot identify the delta in this paragraph.

Comment (smithfarm) [inline doc/architecture.rst]: sorry, there's no delta in the `Data Scrubbing` paragraph - I think it was just a whitespace change. Sorry, fixing.

File: doc/architecture.rst [hunk 2]
```diff
@@ -360,13 +360,15 @@ ability to leverage this computing power leads to several major benefits:
    Ceph Client requests. If a Ceph OSD Daemon is ``down`` and ``in`` the Ceph 
    Storage Cluster, this status may indicate the failure of the Ceph OSD 
    Daemon. If a Ceph OSD Daemon is not running (e.g., it crashes), the Ceph OSD 
-   Daemon cannot notify the Ceph Monitor that it is ``down``. The Ceph Monitor 
-   can ping a Ceph OSD Daemon periodically to ensure that it is running. 
-   However, Ceph also empowers Ceph OSD Daemons to determine if a neighboring 
-   OSD is ``down``, to update the cluster map and to report it to the Ceph 
-   monitor(s). This means that Ceph monitors can remain light weight processes. 
+   Daemon cannot notify the Ceph Monitor that it is ``down``. The OSDs
+   periodically send messages to the Ceph Monitor (``pg stats`` pre-luminous,
```
Comment (tchaikov) [inline doc/architecture.rst]: maybe, s/pg stats/MPGStats/ to be more symmetric ? as `MOSDBeacon` is a message type. but i am fine either way.

Comment (liewegas) [inline doc/architecture.rst]: yeah that's be better.  otherwise lgtm, ok to merge with that change!

**Comment by liewegas** [comment]
Mostly teh second, but this is better: "Normally Ceph OSD Daemons determine if a neighboring OSD is down and to report it to the Ceph monitor(s)."  The mon marknig down due to a missing beacon is a fail safe (if, say, all osds are down), and very slow.

**Comment by smithfarm** [comment]
@liewegas Please check out the commit I just added. I also made the change suggested by @tchaikov.

**Pull request merged**